pkg/scheduler/eventhandlers_test.go

```diff
@@ -104,7 +104,9 @@ func TestSkipPodUpdate(t *testing.T) {
 			expected: false,
 		},
 	}
-	for _, test := range table {
+	for i := range table {
+		test := table[i]
```

Comment:
Why this change? It's functionally identical with one lookup and one allocation more than before.

Reply:
the current violates scopelint check and may cause data race problem as https://github.com/kubernetes/kubernetes/pull/78563#discussion_r289311252

Reply:
FYI: https://github.com/kyoh86/scopelint#whats-this

Reply:
Thanks for the explanation! I wasn't aware of this.